src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/helper.py

```
@@ -106,6 +107,13 @@ def make_request(
                 err=e, message_key="message", default_err=default_err
             )
             raise ExtractorError(msg)
+        except LLMWhispererClientException as e:
+            logger.error(f"LLM Whisperer error: {e}")
+            raise ExtractorError(f"LLM Whisperer error: {e}")
+        except Exception as e:
+            logger.error(f"Adapter error: {e}")
+            raise ExtractorError(f"Adapter error: {e}")
```

```suggestion
```
We shouldn't have such catch all blocks here that catch and re-raise a different error unnecessarily. We can propagate such errors and display a 500 Internal Server Error rather than showing the user a pythonic exception

---

Got it, I'll propagate the error instead of showing the user a pythonic exception